.editorconfig

```diff
@@ -1,6 +1,3 @@
-# This file is for unifying the coding style for different editors and IDEs
-# editorconfig.org
-
```

Comment:
I'd rather keep this comment if possible, to let people know what this file is for 😁